org.cloudcompare.CloudCompare.yaml

```diff
@@ -372,7 +372,8 @@ modules:
       - type: git
         url: https://github.com/CloudCompare/CloudCompare.git
         #tag: v2.13.2
-        branch: version_2.13.X
+        #branch: version_2.13.X
+        commit: d19b039d46f176ff4dd27e2d3498dbd5ee8eb591
```

Comment:
FYI, builds must be reproducible. Therefore you have to point to references with a fixed state, e.g. raw commits or tagged commits.